### PR TITLE
feat: sign up with Google

### PR DESCRIPTION
**Pull request by suryapts007**
This PR links to this issue: #791 
Now users can see a new option to sign up with Google on the signup page.

**Screenshots (Sing In Page):** 
Screenshot 2024-01-24 at 2 17 49 AM
Screenshot 2024-01-24 at 2 18 19 AM

**Screenshots (Sing Up Page):**
Screenshot 2024-01-24 at 2 19 04 AM
Screenshot 2024-01-24 at 2 19 26 AM

### CODE REVIEW

File: apps/web/src/components/forms/signin.tsx [hunk 1]
```diff
@@ -221,7 +221,7 @@ export const SignInForm = ({ className, isGoogleSSOEnabled }: SignInFormProps) =
               onClick={onSignInWithGoogleClick}
             >
               <FcGoogle className="mr-2 h-5 w-5" />
-              Google
+              Sign In with Google
```
Comment (dguyen) [inline apps/web/src/components/forms/signin.tsx:224]: Could you please revert the changes in this file?

**Review by dguyen — CHANGES_REQUESTED** [review]

**Comment by anikdhabal** [comment]
Hey @dguyen in the Pr https://github.com/documenso/documenso/pull/727 I've fixed this one also.

**Comment by dguyen** [comment]
@anikdhabal thanks but I'm not sure that PR is going anywhere anytime soon due to the Apple setup required

**Comment by anikdhabal** [comment]
> @anikdhabal thanks but I'm not sure that PR is going anywhere anytime soon due to the Apple setup required

Yeah, that's true. Okay, then I will make sure to change that PR before merging according to the changes.

File: apps/web/src/components/forms/signin.tsx (hunk 1 above)
Comment (suryapts007) [reply apps/web/src/components/forms/signin.tsx]: yeah sure. Done with it.

Though, would like to know the idea behind keeping it same.

**Review by dguyen — APPROVED** [review]
Looks good to me, thanks for this :)
